### PR TITLE
Add Team page

### PR DESCRIPTION
**Pull request by stephanieinez**
Add new team page to url `/team`

(Issue #13)

### CODE REVIEW

File: src/containers/Team/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const Team = (item) => (
+    <div>
+        <div className="page-background">
+            <Header/>
+            <div className="team-header">
+                Team Carbon Doomsday
+            </div>
+            <div>
+                <div className="team-grid">
+                {teamItems.map((item) => (
```
Comment (grady-lad) [inline src/containers/Team/index.js]: Here we should pass `teamItems` down as props. 

We could also take advantage of [default props](https://facebook.github.io/react/docs/typechecking-with-proptypes.html#default-prop-values)

by doing something like 
````javascript
Team.defaultProps = {
teamItems: teamItems
};
````

And then within the team component pass `teamItems` as props.

The reason for this approach is that if anyone else were to reuse this component and wanted to pass their own team members props they could.

If the `teamItems` prop is not passed to the component then we just use the defaultProp which is teamItems from our constants file. 

So for us we would be using `Team` component without passing it any props

Comment (grady-lad) [reply src/containers/Team/index.js]: It would also be of worth to add `proptypes` to this component

File: src/index.js [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
 import ReactDOM from 'react-dom';
+import {  BrowserRouter, Route, Switch } from 'react-router-dom';
+import { ROOT, TEAM_PAGE } from './routes/index';
 import App from './containers/App';
+import Team from './containers/Team';
 import registerServiceWorker from './registerServiceWorker';
 
 loadScripts();
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render((
```
Comment (grady-lad) [inline src/index.js]: 👍

Comment (grady-lad) [reply src/index.js]: The footer should be updated with the link to the team page, no ?

**Comment by lwm** [comment]
Max stoke.

File: src/index.js (hunk 2 above)
Comment (stephanieinez) [reply src/index.js]: Done!

File: src/containers/App/index.js [hunk 3]
```diff
@@ -238,23 +238,23 @@ class App extends Component {
           <LoadingWrapper loading={loading}>
             <div>
               <InfoColumnHOC
-                rangeType={rangeType}
-                currentPPM={currentPPM}
-                ppmDiff={ppmDiff}
-                ppmPercentDiff={ppmPercentDiff}
-                diffPPMSubHeader={diffPPMSubHeader}
-                diffPercentSubHeader={diffPercentSubHeader}
+                  rangeType={rangeType}
```
[the remark this answers is not in the document]
Comment (stephanieinez) [reply src/containers/App/index.js]: This will be fixed once we merge #45

**Pull request merged**